src/common/legacy_config_opts.h

```diff
@@ -462,6 +462,8 @@ OPTION(mds_debug_subtrees, OPT_BOOL)
 OPTION(mds_kill_mdstable_at, OPT_INT)
 OPTION(mds_kill_export_at, OPT_INT)
 OPTION(mds_kill_import_at, OPT_INT)
+OPTION(mds_kill_dirfrag_export_at, OPT_INT)
+OPTION(mds_kill_dirfrag_import_at, OPT_INT)
```

Comment:
We don't modify this file anymore.

Reply:
Should have mentioned this before but it gave build errors when I left it out of this file.